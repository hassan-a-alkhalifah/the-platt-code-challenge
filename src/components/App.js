import React from 'react';
import './App.css';
import HeaderComponent from './Header/HeaderComponent';
import BodyComponent from './Body/BodyComponent';

function App() {
  return (
    <div className="App">
      <HeaderComponent />
      <BodyComponent />
    </div>
  );
}

export default App;
