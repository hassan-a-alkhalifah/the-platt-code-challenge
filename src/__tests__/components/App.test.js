import React from 'react';
import App from '../../components/App';
import { shallow, configure } from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

describe('<App /> tests', () => {
    it('Snapshot test. Does render', () => {
        const AppComponent = <App />;
        const component = shallow(AppComponent);
        expect(component).toMatchSnapshot();
    });
});