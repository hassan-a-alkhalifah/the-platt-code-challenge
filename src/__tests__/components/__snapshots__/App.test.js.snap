// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> tests Snapshot test. Does render 1`] = `ShallowWrapper {}`;
